source/MaterialXRender/OpenGL/GLTextureHandler.cpp

```diff
@@ -98,26 +110,16 @@ bool GLTextureHandler::acquireImage(std::string& fileName,
         textureLoaded = true;
     }
 
-    if (!textureLoaded)
+    // Create a fallback texture if failed to load
+    if (!textureLoaded && fallbackColor)
```

Comment:
Logic change based on ILM change for a constant filename as input. Now we allow an explicit fallback color which will drive create a solid color image with the same cache key (original file name). This makes it so the client does not need to track another made up cache key name but also now allow for the default color to be passed over if the image does not load (which is more correct then just using black).